HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -404,6 +404,16 @@ def customizeHLTfor44778(process):
         if hasattr(process, modLabel):
             mod = getattr(process, modLabel)
             mod.l1EGCand = cms.InputTag('hltEgammaCandidatesPPOnAA')
+
+def customizeHLTFor45478(process):
+
+    ca_builders = ["CAHitNtupletAlpakaPhase1@alpaka", "alpaka_serial_sync::CAHitNtupletAlpakaPhase1"]
+
+    for ca_builder in ca_builders:
+        for producer_name in producers_by_type(process, ca_builder):
+            producer = getattr(process,producer_name.label_())
+            producer.phiCuts = cms.vint32( 522, 730, 730, 522, 626, 626, 522, 522, 626, 626, 626, 522, 522, 522, 522, 522, 522, 522, 522 , 522, 522)
+
     return process
```

Comment:
This seems to be the return process originally from customizeHLTfor44778() which you 'reassigned' to your added function.